packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1198,6 +1254,17 @@ async function getGCStateFromSnapshot(
     return rootGCState;
 }
 
+function generateSortedGCState(gcState: IGarbageCollectionState): IGarbageCollectionState {
+    const sortableArray: [string, IGarbageCollectionNodeData][] = Object.entries(gcState.gcNodes);
+    sortableArray.sort(([a], [b]) => a.localeCompare(b));
+    const sortedGCState: IGarbageCollectionState = { gcNodes: {} };
+    for (const [nodeId, nodeData] of sortableArray) {
+        nodeData.outboundRoutes.sort();
+        sortedGCState.gcNodes[nodeId] = nodeData;
```

Comment:
```suggestion
        sortedGCState.gcNodes[nodeId] = nodeData.outboundRoutes.sort();
```

Reply:
nodeData.outboundRoutes.sort() returns a reference to the sorted array

Reply:
Hmmm. I thought it returns the sorted array. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort